tests/e2e/kubectl/kubectl.go

```diff
@@ -56,10 +59,17 @@ type Kubectl interface {
 	RawCommand(subCmd string, timeout time.Duration) *executor.CMDResult
 }
 
+// FilenameOption:
+//
+//	kubectl.Filename  // --filename
+//	kubectl.Kustomize // --kustomize
 type ApplyOptions struct {
-	Namespace string
-	Output    string
-	Force     bool
+	Filename       []string
+	FilenameOption FilenameOption
```

Comment:
We already have Filename. Why is it need?

Reply:
FilenameOption is used for --kustomize/--filename options.